prime-router/src/main/kotlin/messages/SettingsMessages.kt

```diff
@@ -0,0 +1,128 @@
+package gov.cdc.prime.router.messages
```

Comment:
The code in this file has been broken out from the `SettingsFacade.kt`. I'm putting these types of REST end-point payload files in a separate directory. Eventually, I'd like to get these files generated from the Swagger yaml.